src/components/ProductsList/ListElement/ListElementStyle.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components'
+
+// language=SCSS prefix=dummy{ suffix=}
+const ListElement = styled.li`
+    font-size: 20px;
+    margin: 20px 0;
+    padding: 20px;
+    border: black 2px solid;
+    height: 100px;
+    
+    &:hover {
```

Comment:
Not a problem with this particular code, but I wonder ... doesn't this approach to define styled components prevent reuse? Or on the contrary - it's designed to prevent reuse?